tests/test_cli.py

```diff
@@ -139,3 +189,62 @@ def test_triton_profile(self, mocker, monkeypatch):
         args = parse_args()
         args.func(args)
         mock_run.assert_called_once_with(["genai-perf", "-m", "add_sub"], check=True)
+
+    @pytest.mark.parametrize("model", ["mock_llm"])
+    def test_triton_metrics(self, model, setup_and_teardown):
+        # Import the Model Repo
+        pid = utils.run_server(repo=MODEL_REPO)
+        setup_and_teardown.pid = pid
+        utils.wait_for_server_ready()
+
+        # Inference the Model
+        self._infer(model, prompt=PROMPT)
+
+        output = ""
+        # Redirect stdout to a buffer to capture the output of the command.
+        with io.StringIO() as buf, redirect_stdout(buf):
+            self._metrics()
+            output = buf.getvalue()
+
+        metrics = json.loads(output)
+
+        # Loop through all loaded models and check for successful inference
+        for loaded_models in metrics["nv_inference_request_success"]["metrics"]:
+            if loaded_models["labels"]["model"] == model:
+                assert loaded_models["value"] > 0
```

Comment:
Can you update this to be a little more strict?
1. assert that the value is `== 0` before inference
2. assert the value is exactly `== 1` after inference

Reply:
My initial thought was to leave it open in the future if multiple inference requests to the same model needed to be tested, but I can definitely make it stricter. And if we want to loosen the `assert` statement in the future, it wouldn't be fairly trivial.